### PR TITLE
add remaining questions

### PR DESCRIPTION
**Pull request by Fbasham**
### Description

Add remaining questions

List of proposed changes:
- added question content and functionality
- added translations
- state is defined in the initalValues prop and accessed via Formik's values prop
- final state can be viewed by console.logging the values in the obSubmit handler in the index.js file

### What to test for/How to test
- navigate the quiz, toggle questions on and off.  The state should be maintained (remembers on/off toggles events).

### Additional Notes
- one of the questions (question 3 in Figma), was removed from the scope of this sprint based on comments raised during the previous demo
- question 2 checkboxes have individual fields in state so as to stay consistent with the other toggle buttons.  This removes the need to flatten any values.  Otherwise MUI checkboxes are grouped in an array.

### CODE REVIEW

File: frontend/src/components/questions/Question4.tsx [hunk 1]
```diff
@@ -1,10 +1,113 @@
...
+          className="my-4 font-display text-base font-bold normal-case"
+          selected={
+            values['retirementTimeframe'] === 'CanadaPT6OrMore'
+          }
+        >
+          <Trans
+            ns="learn"
+            i18nKey="quiz.questions.question-4.option-2"
+            components={{
+              upper: <span className="px-1 underline"></span>,
```
Comment (gregory-j-baker) [inline frontend/src/components/questions/Question4.tsx:72]: According to the canada.ca style guide:

> Use underlining for links only.

https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-2

File: frontend/src/components/questions/Question4.tsx [hunk 2]
```diff
@@ -1,10 +1,113 @@
...
+          className="my-4 font-display text-base font-bold normal-case"
+          selected={
+            values['retirementTimeframe'] === 'CanadaPTLessThan6'
+          }
+        >
+          <Trans
+            ns="learn"
+            i18nKey="quiz.questions.question-4.option-3"
+            components={{
+              upper: <span className="px-1 underline"></span>,
```
Comment (gregory-j-baker) [inline frontend/src/components/questions/Question4.tsx:88]: According to the canada.ca style guide:

> Use underlining for links only.

https://www.canada.ca/en/treasury-board-secretariat/services/government-communications/canada-content-style-guide.html#wp4-2

File: frontend/src/components/questions/Question2.tsx [hunk 3]
```diff
@@ -1,94 +1,131 @@
...
+  const [value, setValue] = React.useState('')
+
+  const handleChange = (
+    event: React.MouseEvent<HTMLElement>,
+    answerId: string
+  ) => {
+    setValue(answerId)
+    setFieldValue(
+      'hasChildren',
+      values['hasChildren'] === answerId ? '' : answerId ?? ''
```
Comment (gregory-j-baker) [inline frontend/src/components/questions/Question2.tsx:28]: **Nitpick:**

```suggestion
      values['hasChildren'] === answerId ? '' : (answerId ?? '')
```

(to make it clear that the in-parenthesis code is evaluated as a single entity)

This occurs in a few places in this PR. Also, if you don't agree and think this should stay as-is, just say so. I'm okay either way.

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]

File: frontend/src/components/questions/Question4.tsx (hunk 2 above)
Comment (Fbasham) [reply frontend/src/components/questions/Question4.tsx]: removed

File: frontend/src/components/questions/Question2.tsx (hunk 3 above)
Comment (Fbasham) [reply frontend/src/components/questions/Question2.tsx]: Sure, readability is important.

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**